aif360/datasets/multiclass_label_dataset.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+
+from aif360.datasets import StructuredDataset
+'''
+Multiclass supports the multiple values in the favorable and unfavorable label's
+'''
+class MulticlassLabelDataset(StructuredDataset):
+    """Base class for all structured datasets with multiclass labels."""
+
+    def __init__(self, favorable_label = [1.], unfavorable_label = [0.], **kwargs):
+        """
+        Args:
+            favorable_label (list): Label value which is considered favorable
+                (i.e. "positive").
+            unfavorable_label (list): Label value which is considered
+                unfavorable (i.e. "negative").
+            **kwargs: StructuredDataset arguments.
+        """
+        self.favorable_label = favorable_label
+        self.unfavorable_label = unfavorable_label
+
+        super(MulticlassLabelDataset, self).__init__(**kwargs)
+
+        fav_label_value = 1.
+        unfav_label_value = 0.
+        #Iterate the labels values in the dataframe and check whether the label value is 
+        # in the favorable list, if it is true then assign the favorable label value as 1
+        # otherwise 0 for unfavourable values
+        for index in range(0,len(self.labels)):

```

Comment:
Could we move this remapping to happen in the `__init__()` function of binary_label_dataset_metric.py and classification_metric.py? That way, the `MulticlassLabelDataset` class can keep the original labels and stay general for the future.

Reply:
@hoffmansc As per the inputs from the last meeting, the code was moved from classification_metric.py to the `MulticlassLabelDataset`, so that the changes related to the multi class will be in single class.

Reply:
Right. I don't want to change the implementations in classification_metric.py, just handle the binarization of the labels there. Otherwise, we lose the information in `MultiClassDataset` labels. Right now, it's effectively no different than a `BinaryLabelDataset` as far as I can tell

Reply:
@hoffmansc Updated the code as per the suggestion. Please review. Thanks!